.prettierrc.cjs

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  trailingComma: 'es5',
+  tabWidth: 2,
```

Comment:
add editorconfig: true
and .editorconfig

Reply:
fixed